corgidrp/detector.py

```diff
@@ -4,6 +4,14 @@
 import corgidrp.data as data
 from scipy import interpolate
 from scipy.ndimage import median_filter
+import glob
```

Comment:
Let's double check these imports to see if all of the are still necessary. I think `os` and `glob` are no longer used?